samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -8,7 +8,7 @@ nRF9160: nRF Cloud MQTT multi-service
    :depth: 2
 
 This sample is a minimal, error tolerant, integrated demonstration of the :ref:`lib_nrf_cloud`, :ref:`lib_location`, and :ref:`lib_at_host` libraries.
-It demonstrates how you can use these libraries together to support Firmware-Over-The-Air (FOTA), Location Services, periodic sensor samples, and more in your `nRF Cloud`_-enabled application.
+It demonstrates how you can use these libraries together to support Firmware-Over-The-Air (FOTA), Location Services, Alert and Log Services, transmit periodic sensor samples, and more in your `nRF Cloud`_-enabled application.
```

Comment:
nit: `transmission of periodic sensor samples` or `periodic sensor sampling`

Reply:
However, the sentence as a whole could probably be made less awkward:

> It demonstrates how you can integrate Firmware-Over-The-Air (FOTA), Location Services, Alert and Log Services, periodic sensor sampling, and more into your `nRF Cloud`_-enabled application.

Reply:
Awkward?  Little o' me?  Ha ha.